app/src/main/java/com/nutomic/syncthingandroid/activities/FirstStartActivity.java

```diff
@@ -191,6 +194,14 @@ private boolean isFirstStart() {
         return mPreferences.getBoolean(Constants.PREF_FIRST_START, true);
     }
 
+    private boolean isNotificationPermissionGranted() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
```

Comment:
```suggestion
        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) {
        return true;
```

Early out. That way you don't need the permission check in the if block.

Reply:
But we do need the else block to check if the permissions are granted or not?

Reply:
But why? we do need else block for other checks anyhow? no?